openpype/plugins/publish/cleanup.py

```diff
@@ -41,8 +41,8 @@ class CleanUp(pyblish.api.InstancePlugin):
     active = True
 
     # Presets
-    paterns = None  # list of regex paterns
-    remove_temp_renders = True
+    paterns = None  # list of regex patterns
```

Comment:
I wanted to fix the typo on the variable as well but this would break backwards compatibility